Compiler warning on linux about edge_end in GetCell

```
pyvoronoi/voronoi.cpp: In member function ‘c_Cell VoronoiDiagram::GetCell(long long int)’:
pyvoronoi/voronoi.cpp:163:35: warning: variable ‘edge_end’ set but not used [-Wunused-but-set-variable]
  163 |                         long long edge_end = -1;
```

Looks like a simple fix, I think it is indeed unused.

--- a/pyvoronoi/voronoi.cpp
+++ b/pyvoronoi/voronoi.cpp
@@ -160,14 +160,9 @@ c_Cell VoronoiDiagram::GetCell(long long index)
 				is_open = true;
 
 			long long edge_start = -1;
-			long long edge_end = -1;
 
 			if (edge->vertex0() != NULL){
 				edge_start = map_vertices_to_indexes[edge->vertex0()];
-			}
-
-			if (edge->vertex1() != NULL){
-				edge_end = map_vertices_to_indexes[edge->vertex1()];
 			}
 
 			long vertices_count = vertex_identifiers.size();

